django-stubs/db/models/sql/compiler.pyi

```diff
@@ -13,9 +13,9 @@ from django.db.models.sql.subqueries import AggregateQuery, DeleteQuery, InsertQ
 from django.utils.functional import cached_property
 from typing_extensions import TypeAlias
 
-_ParamT: TypeAlias = str | int
+_ParamT: TypeAlias = Any
```

Comment:
This whole module must be revisited. There lots of incorrect annotations there :(

Reply:
Copy that, thank you! Then I'll close the PR for now)